doc/ug/setup.tex

```diff
@@ -127,7 +127,7 @@ \section{\texttt{setmd}: Setting global variables.}
 \item[periodicity] (bool[3]) Specifies periodicity for the three
   directions. If the feature PARTIAL_PERIODIC is set, this variable
   can be set to (1,1,1) or (0,0,0) at the moment.  If not it is
```

Comment:
Ehm, just looking at the diff, I noted that line. That is wrong as well. I am pretty sure that MMM2D e.g. can be used only with 1, 1, 0 and MMM1D with 1, 0, 0, which according to this would mean they can't be used at all :-)